Sources/RingPublishingTracking/Private/Decorator/Generic/Models/PaidEventUserId.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  PaidEventUserId.swift
+//  RingPublishingTracking
+//
+//  Created by Bernard Bijoch on 04/09/2024.
+//
+
+import Foundation
+
+struct PaidEventUserId: Encodable {
+
+    enum CodingKeys: String, CodingKey {
+        case fakeUserId = "fake_user_id"
+        case realUserId = "real_user_id"
+    }
+
+    let fakeUserId: String?
```

Comment:
to ma tylko sens jak są dwie wartości

Reply:
@adamszeremeta ten obiekt wykorzystywany jest tez przy createPurchaseEvent, gdzie idzie tylko fake_user_id

Reply:
ale tam ma Bernard to podane jawnie jako
 temporaryUserId: String?
 
 bo potem to nie wymusza w tym zdarzeniu z przepisaniem usera podania dwoch wartosci.. chyba ze przez konstruktor to wymusimy a zostanie do uzytku wewnetrznego opcjonalne to ok